packages/inter-protocol/src/proposals/add-auction.js

```diff
@@ -9,40 +9,51 @@ const trace = makeTracer('NewAuction', true);
 /**
  * @typedef {PromiseSpaceOf<{
  *   auctionUpgradeNewInstance: Instance;
+ *   newContractGovBundleId: string;
```

Comment:
hm. "new" with respect to what? Are we certain we'll never do this again? If not, what would we call it next time?